test/unit/core/error_helpers.js

```diff
@@ -177,6 +177,28 @@ suite('Error Helpers', function() {
         'ValidationError type is correct'
       );
     });
+
+    testUnMinified('line: null string given', function() {
+      let err = assert.throws(function() {
+        p5._validateParameters('line', [1, 2, 4, 'null']);
+      }, p5.ValidationError);
+      assert.strictEqual(
+        err.type,
+        'WRONG_TYPE',
+        'ValidationError type is correct'
+      );
+    });
+
+    testUnMinified('line: infinite value given', function() {
+      let err = assert.throws(function() {
+        p5._validateParameters('line', [1, 2, 4, Infinity]);
+      }, p5.ValidationError);
+      assert.strictEqual(
+        err.type,
+        'WRONG_TYPE',
+        'ValidationError type is correct'
+      );
+    });
```

Comment:
CI is failing on this test case, I don't think we need to cover for `Infinity` as well since `Infinity` is a valid number that can be useful in certain circumstances (eg. `min()` `max()`)

Reply:
My bad, that makes sense! I just pushed a commit to slightly change that test to check for NaN instead of Infinity since that should return a ValidationError and checking for NaN was originally the main objective.